File: demo_glm/grouping1.R
suppressMessages(library(PivotalR))
gp <- db.connect(port=16526, dbname='madlib', host='127.0.0.1', user='gpadmin', verbose=F, quick=T) # 1, Greenplum database
x <- db.data.frame('madlibtestdata.dt_abalone', verbose=F)

f1 <- madlib.glm(rings ~ . - id | sex, data = x, family = poisson(log))
f1

File: demo_glm/grouping1.go
package main

import (
	"fmt"
	"os/exec"
)

func main() {
	out, err := exec.Command("Rscript", "demo_glm/grouping1.R").Output()
	if err != nil {
		fmt.Println(err)
		return
	}
	fmt.Println(string(out))
}

/* Show in the slide
// Start OMIT

f1 <- maldib.glm(rings ~ . - id | sex, data = x, family = poisson(log))
f1

// End OMIT
*/
